onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlDFT.h

```diff
@@ -183,22 +232,96 @@ class GpuDFTOperator : public WRL::Base<IMLOperatorKernel>
         ORT_THROW_IF_FAILED(context->GetInputEdgeDescription(0, &edgeDesc));
         assert(edgeDesc.edgeType == MLOperatorEdgeType::Tensor);
 
-        PrepareGpuResources(edgeDesc.tensorDataType);
+        PrepareStockhamFFT(edgeDesc.tensorDataType);
+        PrepareBluesteinZChirp(edgeDesc.tensorDataType);
+    }
+
+    void PrepareBluesteinZChirp(MLOperatorTensorDataType dataType)
+    {
+        // Compute root signature.
+        const int uavCount = 2; // 2 outputs: chirp, and the reflected chirp conjugate (B)
+        std::vector<CD3DX12_ROOT_PARAMETER1> rootParameters;
+        rootParameters.resize(uavCount + 1);
+
+        for (uint32_t i = 0; i < uavCount; i++)
+        {
+            rootParameters[i].InitAsUnorderedAccessView(i);
+        }
+
+        // cbuffer Constants
```

Comment:
>cbuffer Constants [](http://example.com/codeflow?start=11&length=17)

```suggestion
// cbuffer Constants // BluesteinZChirpShaderConstants
```

Just so the correspondence is immediately clear.